usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -102,6 +105,23 @@
         "column_names": list(TRANSACTION_FABS_COLUMNS),
         "tsvectors": None,
     },
+    "published_fabs": {
+        "model": SourceAssistanceTransaction,
+        "is_from_broker": False,
+        "source_table": "published_fabs",
+        "source_database": "raw",
+        "destination_database": "raw",
+        "swap_table": None,
+        "swap_schema": None,
+        "partition_column": "detached_award_procurement_id",
```

Comment:
I believe this should be `published_fabs_id`

Reply:
addressed